src/rdb.c

```diff
@@ -3391,6 +3392,8 @@ int rdbLoadRioWithLoadingCtx(rio *rdb, int rdbflags, rdbSaveInfo *rsi, rdbLoadin
                 val = setExpire(NULL, db, &keyobj, expiretime);
             }
 
+            dbTrackKeyWithVolaItemsIfNeeded(db, val);
```

Comment:
can you place a comment here that this is needed because we did not know the DB context before when we loaded the object?